ah-med/MaintenanceTracker
# chore(database): setup database

**What this PR does**
 - setup database for data persistence

 **Tasks completed**
- add node-postgres database to dependencies
- add dotenv to dependenies
- create config file to hold environment variables
- create .env.default file to hold environment variables template

 **How this can be manually tested**
 - check the Package.json file for the added dependencies
 - check config directory for the database configurations
 - check .env.default file for environment variables template

 **Relevant pivotal tracker story**
 [#157806637](https://www.pivotaltracker.com/story/show/157806637)
